docs/index.md

```diff
@@ -0,0 +1,33 @@
+# Radar Software Toolkit (RST)
+
+RST is an open source software package that allows researchers to analyze, model, and quickly visualize the SuperDARN project data.
+
+## RST source code 
+
+Currently, our software is hosted on the [SuperDARN GitHub](https://github.com/SuperDARN/rst) repository. 
+If you cannot access GitHub then you can download newest release on SuperDARN VT  [website](http://vt.superdarn.org/tiki-index.php?page=software).
+
+If you have any question or concerns please submit an Issue on the SuperDARN RST repository or email: Data Analysis Working Group. 
```

Comment:
There's no email address provided here. Perhaps remove this part of the sentence?

Reply:
fixed and added a link to creating a new issue on RST.